tests/test_build/test_build.py

```diff
@@ -60,6 +60,18 @@ def test_autofunction_example(self):
             '      // This is the example.\n'
             '      exampleTag();\n')
 
+    def test_autofunction_destructured_params(self):
+        """Make shure that all documented params appears in the function
+        definition."""
+        self._file_contents_eq(
+            'autofunction_destructured_params',
+            u'destructuredParams(p1, p2)\n\n'
```

Comment:
Remove all but first "u". (Unicode strings which collapse into non-unicode ones at the tokenizer level yield unicode strings.)

Reply:
You are right, I removed useless unicode literals :)